ui/src/components/ExportFab.js

```diff
@@ -5,16 +5,25 @@ import "./ExportFab.css";
 import CloudUpload from "@material-ui/icons/CloudUpload";
 import Button from "@material-ui/core/Button";
 import Tooltip from "@material-ui/core/Tooltip";
+import TextField from '@material-ui/core/TextField';
```

Comment:
Can be a separate PR -- Can you make the dialog only show up if something is selected?